charts/manticoresearch/templates/manticore-balancer.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: {{ include "manticoresearch.fullname" . }}-balancer
   labels:
     name: {{ include "manticoresearch.fullname" . }}-balancer
+    release: {{ .Release.Name }}
```

Comment:
This label is duplicated in `include "manticoresearch.labels"`. Remove it please

Reply:
Done